packages/featuregates/bundle/config/upstream/default-featuregate-cr.yaml

```diff
@@ -0,0 +1,10 @@
+#@ load("@ytt:data", "data")
+
+#@ if data.values.TKG_CLUSTER_ROLE != "workload" and data.values.PROVIDER_TYPE != "tkg-service-vsphere":
```

Comment:
I think we don't need to check for cluster type here, as featuregates package is a management package and is installed only on the management cluster. Also, how are we passing the `PROVIDER_TYPE` value to featuregates package?